src/postgres/event-handlers/RootPostCreated.ts

```diff
@@ -0,0 +1,21 @@
+import { PostId, Post } from '@subsocial/types/substrate/interfaces/subsocial';
+import { insertPostFollower } from '../insert-follower';
+import { insertActivityForPost } from '../insert-activity';
+import { fillNewsFeedWithAccountFollowers, fillNewsFeedWithBlogFollowers } from '../fill-activity';
+import { SubstrateEvent } from '../../substrate/types';
+
+
+export const onRootCreated = async (eventAction: SubstrateEvent, post: Post) => {
+  const { data } = eventAction;
```

Comment:
Fix this copypasta on all post events handlers:

```typescript
const { data } = eventAction;
const follower = data[0].toString();
const postId = data[1] as PostId;
```

it should be something like this everywhere:

```typescript
const { author, postId } = parsePostEvent(eventAction);
```